app/controllers/topics_controller.rb

```diff
@@ -397,7 +397,13 @@ def update
       bypass_bump = should_bypass_bump?(changes)
 
       first_post = topic.ordered_posts.first
-      success = PostRevisor.new(first_post, topic).revise!(current_user, changes, validate_post: false, bypass_bump: bypass_bump)
+      success = PostRevisor.new(first_post, topic).revise!(
+        current_user,
+        changes,
+        validate_post: false,
+        bypass_bump: bypass_bump,
+        skip_advance_draft_seq: [true, "true"].include?(params[:skip_advance_draft_seq])
```

Comment:
I like the name `keep_draft` better, or perhaps (`keep_existing_draft`) at least the API will advertise slightly better what we are trying to do here. draft_seq feels like an implementation detail that we don't want to include in our REST API.

Reply:
That's a much better name, thanks! I've made the change.